docs/docs/troubleshooting/hardware-issues.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Hardware Issues
+sidebar_label: Hardware Issues
+description: Troubleshooting common hardware issues.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+export const TesterTabs = (props) => (
+    <Tabs
+    groupId="testers"
+    defaultValue="promicro"
+    values={[
+      { label: "Pro Micro Compatible Board", value: "promicro" },
+      { label: "Seeed Studio XIAO Compatible Board", value: "xiao" },
+    ]}
+  >
+    {/* eslint-disable-next-line */}
+    {props.children}
+  </Tabs>
+
+);
+
+export const PinTabs = (props) => (
+    <Tabs
+    groupId="pin-approaches"
+    defaultValue="shield"
+    values={[
+      { label: "Tester Shield", value: "shield" },
+      { label: "Multimeter", value: "multimeter" },
+      { label: "Visual Inspection", value: "visual" },
+    ]}
+  >
+    {/* eslint-disable-next-line */}
+    {props.children}
+  </Tabs>
+
+);
+
+## Pin Connections
+
+### Identifying Issues
+
+There are three general approaches we recommend for identifying potential issues with microcontroller pins.
```

Comment:
I think this can use a bit of expanding. At a high level, I don't think this is about identifying issues with microcontroller pins, the goal is to identify issues with the PCB in general. e.g. the multimeter tab clearly talks about checking diode connections, etc.

Reply:
I think what would be good to document here is also the distinction made in https://docs.splitkb.com/hc/en-us/articles/360010552339-One-or-more-keys-don-t-work, i.e. one key not working vs. a row/column. This way the user can know where exactly to check with the multimeter/visual inspection approaches.

Reply:
Tried expanding on it a bit.